electricpy/visu.py

```diff
@@ -321,4 +325,209 @@ def get_output_power(self):
         # low slip
         p_x_2 = center_x - self.radius * _np.cos(beta_1)
         p_y_2 = center_y + self.radius * _np.sin(beta_1)
-        return [p_x_1, p_y_1], [p_x_2, p_y_2]
\ No newline at end of file
+        return [p_x_1, p_y_1], [p_x_2, p_y_2]
+
+class PowerCircle:
+    r"""
+    Plot Power Circle Diagram of Transmission System.
+
+    This class is designed to plot the power circle diagram of a transmission
+    system both sending and reciving ends.
+
+    Parameters
+    ----------
+    type: str "sending" or "receiving"
```

Comment:
This can be simplified to:

```python
type:    ["sending", "receiving"]
```

To indicate that the input must be one of the two string parameters specified.

Additionally, use of the variable-name `type` should be avoided since that will overload the `type()` function provided in Python's builtins. Perhaps the name could be changed to `flow_type` or something similar?

Admittedly, I'm a terrible offender of overloading the `type` function, but I'd like to move towards a more consistent format for the project as a whole, i.e., help us all be better.

So...

- [ ] Change name to avoid overloading `type`
- [ ] Simplify description to `  ["sending", "receiving"]`